code/modules/mob/living/carbon/human/species_types/roguetown/roguetown_species.dm

```diff
@@ -18,7 +18,7 @@
 #define REGEX_ENDWORD 3
 #define REGEX_ANY 4
 
-/datum/species/proc/handle_speech(datum/source, mob/speech_args)
+/datum/species/proc/handle_speech(datum/source, list/speech_args)
```

Comment:
This is used like a list here, is declared as a list in most declarations of handle_speech and is actually a lust(I checked via the debugger. I didn't check the two preceding cases in tgcode but I'm pretty sure the same applies.